src/components/TimeInput/__stories__/Controlled.stories.tsx

```diff
@@ -0,0 +1,24 @@
+import { Story } from '@storybook/react'
+import TimeInput from '..'
+import ControlValue from '../../../__stories__/components/ControlValue'
+import { SelectOption } from '../../RichSelect'
+
+export const Controlled: Story = () => (
+  <ControlValue<SelectOption> value={{ label: '03:30', value: '03:30' }}>
+      {({ value, onChange }) => (
+        <TimeInput
+          name="timeinput-test-controlled"
+          onChange={onChange}
+          placeholder="Time"
+          value={value}
+        />
+      )}
+    </ControlValue>
+)
```

Comment:
Do not use `ContolValue` but useState instead. It will show a snippet on how to make it work with hook state form react instead of one of our custom components.